sktime/distances/mpdist.py

```diff
@@ -0,0 +1,275 @@
+import numpy as np
+import pandas as pd
+
+def sliding_dot_products(q, t, q_len, t_len):
```

Comment:
For performance purposes it may be better to use a numpy array rather than panda series

Reply:
Actually I think your docstrings are just out of date, appears that you're already using np arrays, right?